website/source/api/system/metrics.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "api"
+page_title: "/sys/metrics - HTTP API"
+sidebar_title: "<code>/sys/metrics</code>"
+sidebar_current: "api-http-system-metrics"
+description: |-
+  The `/sys/metrics` endpoint is used to get telemetry metrics for Vault.
+---
+
+# `/sys/metrics`
+
+The `/sys/metrics` endpoint is used to get telemetry metrics for Vault.
+
+## Read Telemetry Metrics
+
+This endpoint returns the telemetry metrics for Vault. It can be used by metrics
+collections systems like [Prometheus](https://prometheus.io) that use a pull
+model for metrics collection.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `GET`    | `/sys/metrics`               | `000 application/json` |
```

Comment:
The "Produces" column can be removed. We just today made this change broadly (https://github.com/hashicorp/vault/commit/d096f62d8dd49836bf0500bd33942f7dde85f5df).

Reply:
Fixed